src/routes/(blog-article)/how-to-setup-the-development-environment/+page.md

```diff
@@ -56,16 +56,13 @@ You do not need to setup all services to contribute to the Torrust Index, but in
 
 Setting up the development environment requires to setup the three main services.
 
-First at all, we need to clone the repositories of the three services. We will clone them in a temporarily folder so you can follow this guide.
+First at all, we need to create the Tmp and torrust folders where we will store the repositories of our services.
```

Comment:
Suggestion: 

"First, we need to create the folder (in this example, a temp folder) where we will store the repositories of our services."